src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+    list-style: none;
+  }
+
+  &__item:last-of-type {
+    margin: 0;
+  }
+
+  &__link {
+    display: inline-block;
+    color: $darkblue;
+    font-weight: 500;
+    text-decoration: none;
+    line-height: 60px;
```

Comment:
As for me, better to use paddings, line-height works too, but link height should depend on font-size